### DISCUSSION TITLE
How do you change size of the pages on the sidebar

### DISCUSSION THREAD

**Discussion by lenicyl** [xournalpp/xournalpp#3398] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/73347143/133415413-0b740230-a960-4b94-bf8b-1ebffd7e4fba.png)

the pages on the sidebar are a bit too small for me, i want to increase the size of the pages (as demonstrated by the annotion) 
i also wanted to ask how i could increase the ui size ?

**Comment by LittleHuba** [comment, marked as answer]
>the pages on the sidebar are a bit too small for me, i want to increase the size of the pages (as demonstrated by the annotation)

See #3384 

> i also wanted to ask how i could increase the ui size ?

You may be able to use `GDK_DPI_SCALE` for this.

```
export GDK_DPI_SCALE=1.5; xournalpp
```
